src/index.js

```diff
@@ -74,3 +76,13 @@ process.on('SIGINT', async () => {
     process.exit(1);
   }
 });
+
+if (process.env.CRON && process.env.NOTIFY_METHOD) {
+  new CronJob.CronJob(
```

Comment:
If there are multiple instances running the web server, this may cause the cron job being run once for each instance / process.

I would suggest we just implement a NodeJS script that can be run using command line. On normal production environment, the devOp can use ordinary cron tab to trigger the job; on our production environment on Heroku, we can use [Heroku scheduler](https://devcenter.heroku.com/articles/scheduler).

In this case, we won't need `CronJob` library; we just need to make sure the script works when run on CLI, just like the [hyperlink cleaning job](https://github.com/cofacts/rumors-api/blob/master/src/scripts/cleanupUrls.js#L136-L138) in rumors-api.